RELEASE/scripts/autoscend/auto_acquire.ash

```diff
@@ -478,7 +478,8 @@ boolean auto_hermit(int amt, item it)
 		return hermit(amt, it);
 	}
 	int initial = item_amount(it);
-	catch hermit(amt, it);
+	//store the result to avoid an error being thrown
```

Comment:
does this actually work?

also I discovered we can use
`cli_execute("hermit * clover");`
to get all remaining clovers left today without throwing an error if we already got them today.